core/src/Platforms/Android/SystemWebview/AuthenticationActivity.cs

```diff
@@ -127,15 +127,23 @@ protected override void OnResume()
                 string chromePackage = GetChromePackage();
                 if (string.IsNullOrEmpty(chromePackage))
                 {
-                    throw new MsalClientException(MsalClientException.ChromeNotInstalledError,
-                        "Chrome is not installed on the device, cannot proceed with authentication");
+                    const string chromeNotInstalledMessage = " Chrome is not installed on the device, cannot proceed with authentication";
+                    throw new MsalClientException(MsalClientException.ChromeNotInstalledError, chromeNotInstalledMessage);
```

Comment:
seems like we are not passing in the ex as the inner exception. this can be fine if it doesn't bring value to the developer. if intentional this is all great.